src/_includes/docs/install/reqs/macos/set-path.md

```diff
@@ -0,0 +1,39 @@
+{% assign terminal=include.terminal %}
+{% assign target = include.target %}
+
+### Add Flutter to your `PATH`
+{:.no_toc}
+
+To run Flutter commands in {{terminal}},
+You need to add Flutter to the `PATH` environment variable.
+THis section covers the permanent addition of the path to the
+Flutter to your `PATH` environment variable.
+This guide presumes that you are using the latest default shell, `zsh`.
+This section adds Flutter to the `PATH` using the `.zshenv` file.
+Z shell uses this file only for environment variables and loads its
+contents before any other configuration file.
+
+1. To add Flutter to your `PATH` in your `.zshenv` file,
+   run the following command.
+
+    ```terminal
+    $ cat <<EOF >> ~/.zshenv
+    # Configure Flutter
+
+    export FLUTTER_HOME=\$HOME/Projects/flutter
+    if [[ \$PATH != *"\$FLUTTER_HOME"* ]]
+    then
+        export PATH=\$FLUTTER_HOME/bin:\$PATH
+    fi
+    EOF
```

Comment:
Same comment as above, this code is overcomplicated.

Reply:
It's hard to read, I would just describe the code they need to add to their shell initialization (whether it's Zsh, Bash, or something else)

Reply:
That's the thing: It should be `zsh` across the board at this version of the OS. I don't want to assume a particular level of knowledge or familiarity with all areas involved.

Reply:
drive by comment, @atsansone you're right, users will copy paste shell snippets to their config. also, many tools auto append. this leads to users having very long, incomprehensible config files and obscure runtime shell environments that we are not easy for my team to debug.